worker/src/index.js

```diff
@@ -0,0 +1,46 @@
+const Config = require('../config.json');
+const rp = require('request-promise');
+const rabbit = require('amqplib').connect(Config.RABBIT_URL);
+
+async function consumeTasks() {
+  try {
+    const conn = await rabbit;
+    const channel = await conn.createChannel();
+    await channel.assertQueue('bookmarks');
+
+    channel.consume('bookmarks', (bookmark) => {
+      checkBookmark(JSON.parse(bookmark.content.toString()));
+      channel.ack(bookmark);
+    });
+  } catch (err) {
+    console.warn(err);
+    // keep going
+    consumeTasks();
+  }
+}
+
+async function checkBookmark(bookmark) {
+  console.log(bookmark);
+  rp(bookmark.url)
```

Comment:
Do you want to be returning this promise so that the function can be `await`ed?  `retun rp()`?